onnxruntime/python/tools/kernel_explorer/kernels/vector_add_kernel.h

```diff
@@ -3,15 +3,20 @@
 
 #pragma once
 
-#include "device_array.h"
-#include "operator.h"
+#include <hip/hip_runtime.h>
+#include "python/tools/kernel_explorer/device_array.h"
+#include "python/tools/kernel_explorer/operator.h"
+#include "contrib_ops/rocm/bert/util.h"
+
+using onnxruntime::contrib::rocm::CeilingDivision;
```

Comment:
Better avoid these type of `using` in header files.
But since we are not exposing these to production code and let keep as is and clean it up latter.

Reply:
This was suggested by linter to use using declaration instead of directive. But thanks and yes, we can improve it later.